Trine.Analyzer/BracketAnalyzer.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace Trine.Analyzer
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    public class BracketAnalyzer : DiagnosticAnalyzer
+    {
+        public const string DiagnosticId = "TRINE03";
+
+        private static DiagnosticDescriptor Rule = new DiagnosticDescriptor(DiagnosticId, "Missing brackets", "Missing brackets", "Category", DiagnosticSeverity.Warning, isEnabledByDefault: true);
```

Comment:
A rule that break long lines would be nice :D
```suggestion
        private static DiagnosticDescriptor Rule = new DiagnosticDescriptor(
        DiagnosticId, "Missing brackets", 
        "Missing brackets", "Category", 
        DiagnosticSeverity.Warning, 
        isEnabledByDefault: true);
```